sprint9/tema2.js

```diff
@@ -0,0 +1,15 @@
+const getNewObjectDeleteProp = (obj, properties) => {
+    const newObject = {...obj};
+        properties.forEach(property => {
```

Comment:
optional - identare:
randurile astea 3-5 au un tab in plus (recomand sa stergi 4 spatii sa fie aliniate cu randul 2)